README.md

```diff
@@ -31,17 +31,45 @@ Build the plugin by running make in repo:
 ```
 $ make
 ```
-This builds the plugin in `/build/rootfs`
+This builds the plugin in `./build`
 
 ### Configuration and Usage
-* Set up the [snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started)
-* Ensure `$SNAP_PATH` is exported
-`export SNAP_PATH=$GOPATH/src/github.com/intelsdi-x/snap/build`
+* Set up the [Snap framework](https://github.com/intelsdi-x/snap#getting-started)
 
 ## Documentation
 [Moving Average](https://en.wikipedia.org/wiki/Moving_average)
 
 ### Examples
+Example running psutil plugin, movingaverage processor, and writing data into an csv file.
```

Comment:
Are you sure that is a csv file? ` writing data into a file` should be enough, in my opinion